api/v1alpha1/cluster_types.go

```diff
@@ -86,6 +86,14 @@ type CrdbClusterSpec struct {
 	// (Required) Database disk storage configuration
 	// +required
 	DataStore Volume `json:"dataStore,omitempty"`
+	// (Optional) Additional custom resource labels
```

Comment:
Could you note where these labels will appear? Are they on the statefulset and pods?

Reply:
Hi @chrisseto ,
It should appear statefulset, pods, etc.
But currently in applies only to ss. I've updated both common and selector functions in labels.go with merge labels passed from crd. Could you please take a look and point what am I missing here? TIA